Dockerfile.release

```diff
@@ -0,0 +1,47 @@
+#---------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+#---------------------------------------------------------------------------------------------
+
+# This Dockerfile is for releases of this CLI.
+#   How to build:
+#    1. Create a temp directory that only contains this Dockerfile.
+#    2. Run 'docker build' with this Dockerfile file once the latest package has been published to PyPI
+#       To create a Docker image for this release.
+#       When tagging this Docker image, choose an appropriate version number.
+#         e.g.: 'sudo docker build -f Dockerfile.release -t azuresdk/azure-cli-python:<version> .'
+#    3. Push the image to the registry,
+#         e.g.: 'docker push azuresdk/azure-cli-python:<version>'
+
+FROM python:3.5.2-alpine
+
+# pip wheel - required for CLI packaging
+# jmespath-terminal - we include jpterm as a useful tool
+RUN pip install --upgrade pip wheel jmespath-terminal
+# bash gcc openssl-dev libffi-dev musl-dev - dependencies required for CLI
+# jq - we include jq as a useful tool
+# openssh - included for ssh-keygen
+# ca-certificates 
+# wget - required for installing jp
```

Comment:
yes it's to explain why we install what we install for future maintenance purposese.